media-libs/mlt/mlt-7.34.1.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..14} )
+inherit python-single-r1 cmake flag-o-matic
+
+DESCRIPTION="Open source multimedia framework for television broadcasting"
+HOMEPAGE="https://www.mltframework.org/"
+SRC_URI="https://github.com/mltframework/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0/7"
+KEYWORDS="amd64 arm64 ~loong ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
```

Comment:
I didn't notice this first, but you can't straight-to-stable bump. All KEYWORDS back to ~arch please.

Did you look at the upstream Changelogs for this bump?